src/main/scala/zio/ftp/Ftp.scala

```diff
@@ -0,0 +1,111 @@
+package zio.ftp
+
+import java.io.{ IOException, InputStream }
+
+import org.apache.commons.net.ftp.{ FTP, FTPClient => JFTPClient, FTPSClient => JFTPSClient }
+import zio.blocking.{ Blocking, effectBlocking }
+import zio.ftp.Settings.FtpSettings
+import zio.stream.{ Stream, ZStream, ZStreamChunk }
+import zio.{ IO, Task, ZIO, ZManaged }
+
+final class Ftp(unsafeClient: JFTPClient) extends FtpClient[JFTPClient] {
```

Comment:
Users can make these, I'd use stricter access control, such as `private` constructor here, or maybe `private[ftp]` for package-private.